packages/features/ee/workflows/lib/reminders/emailReminderManager.ts

```diff
@@ -258,12 +207,8 @@ export const scheduleEmailReminder = async (args: scheduleEmailReminderArgs) =>
   const batchId = await getBatchId();
 
   function sendEmail(data: Partial<MailData>, triggerEvent?: WorkflowTriggerEvents) {
-    const status: ParticipationStatus =
-      triggerEvent === WorkflowTriggerEvents.AFTER_EVENT
-        ? "COMPLETED"
```

Comment:
`COMPLETED` is not part of the ICS standard. Most calendars handle bookings in the past on their own